server/app/views/admin/programs/ProgramIndexView.java

```diff
@@ -387,6 +387,9 @@ private LiTag renderPublishModalProgramItem(
       ProgramDefinition program, ImmutableList<Long> universalQuestionIds) {
     String visibilityText = " ";
     switch (program.displayMode()) {
+      case DISABLED:
```

Comment:
If the flag is off, `Disabled` display mode won't be provided an option to the civiform admins.
https://github.com/civiform/civiform/blob/347728fb9ea161ab158bcb8b3ba4d54682116cb5/server/app/views/admin/programs/ProgramFormBuilder.java#L191

Reply:
I'm thinking of the situation like I outlined in your last PR where an admin turns the flag on, marks some programs as disabled, then we find a bug and they have to turn it off. In that case I think the pop-up would still show "Hidden from applicants and trusted intermediaries" which would be incorrect. But there might be a way to handle this deeper in the logic. Writing this now, it's making me wonder what will actually happen with programs marked "Disabled" if we have to toggle the flag off after turning it on. My guess is they should be tagged with the next most restrictive option (probably "Hide from applicants. Only individuals with the unique program link can access this program") but I think this is another product question. Will you ask Isabel and Sijia how they would want to handle this?

I can see a solution where if the flag is off we change the display mode on any programs marked "Disabled", so I think it's actually fine to merge this as is for now without wrapping it in a feature flag. It's also possible I'm overthinking this 😅 . @dkatzz what do you think?

Reply:
Yeah this is a good question - maybe we could add some sort of durable job to mark anything disabled with the next restrictive option if the flag is off? But that may be overkill and probably worth a larger discussion before doing. In the meantime, I think it can't hurt to add an if statement within this condition to say:

if (flagEnabled) {
visibilityText = " (Hidden from applicants and Trusted Intermediaries) ";
}
break;

That way, it just doesn't show that text if the flag is off.

Reply:
Okay, I added a feature flag check to this case :)